src/components/layout/footer-links.tsx

```diff
@@ -12,9 +13,9 @@ export default function FooterLinks({
   return (
     <div className="flex grow flex-col gap-y-12px lg:gap-y-10px lg:pt-10px">
       <div className="hidden lg:block">
-        <Link href={"/newsletter"}>
+        <Link href={""}>
```

Comment:
on peut enlever le link, et il faudrait mettre le onClick sur la textbox pour que la zone entière soit clickable (il faudra ajouter une prop onClick au component textBox)

Reply:
Done, also had to add the cursor-pointer parameter to the TextBox to get the right hovering, tell me what you think